.github/workflows/label-zero-impact.js

```diff
@@ -11,6 +11,7 @@ const zeroImpactDirs = [
   'LICENSE',
   'codecov.yaml',
   '.gitignore',
+  'package-lock.json',
```

Comment:
@chrischrischris mentioned graybox is also zero impact, however we currently only allow full directories rather than paths to be checked.
Wonder if we should change that?

Reply:
I asked for some details on that. If it's confirmed that Graybox needs to be part of the exception list, we can open a quick story to make the change.